packages/next/test/fixtures/00-trailing-slash-middleware-add/middleware.js

```diff
@@ -0,0 +1,5 @@
+
+      export default function middleware() {
```

Comment:
nit: lint

Reply:
oh this file is written dynamically so can be removed actually.